compiler/atomic.go

```diff
@@ -14,7 +14,7 @@ import (
 func (b *builder) createAtomicOp(call *ssa.CallCommon) (llvm.Value, bool) {
 	name := call.Value.(*ssa.Function).Name()
 	switch name {
-	case "AddInt32", "AddInt64", "AddUint32", "AddUint64", "AddUintptr":
```

Comment:
@kyleconroy I think you want to keep the old ones here too, not remove them.  If you do that, I don't _think_ you'll need to pull in `atomic/value.go` as well.

Reply:
I think you have to copy it all over because of the changes to `loader/goroot.go`

Reply:
Ah, when I was reading the code I thought it was maybe file by file (not package by package).

Reply:
But I think the maintainers are going to want to keep the original (non llvm_) AST rewrites anyway (at least for now) to 100% preserve legacy behavior, but let's see what they say.